README.md

```diff
@@ -1,34 +1,44 @@
-| `example1` | :heavy_check_mark:       | Lorem ipsum dolor sit amet, consetetur sadipscing elitr,                                           |
-| `example2` | :heavy_multiplication_x: | Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. |
-| `example3` | `True`                   | Stet clita kasd gubergren                                                                          |
-| `example4` | `5`                      | No sea takimata sanctus est Lorem ipsum dolor sit amet.                                            |
+| Name                                |                               Required/Default                                | Description                                                                                                                        |
+| ----------------------------------- | :---------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `node_exporter_version`             |                                   `latest`                                    | The version of node exporter that should be installed. If empty or set to `latest` the variable will be set to the newest release. |
+| `node_exporter_web_listen_address`  |                                `0.0.0.0:9100`                                 | The IP address the service should bind to.                                                                                         |
+| `node_exporter_web_telemetry_path`  |                                  `/metrics`                                   | Path where the metrics should be served.                                                                                           |
+| `node_exporter_textfile_dir`        |                           `/var/lib/node_exporter`                            | Directory for the textfile exporter.                                                                                               |
+| `node_exporter_tls_server_config`   |                                     `{}`                                      | Config for the TLS feature.                                                                                                        |
+| `node_exporter_basic_auth_users`    |                                     `{}`                                      | List of users and BCRYPT password hashes for basic auth.                                                                           |
+| `node_exporter_enabled_collectors`  | `["systemd", {"textfile": {"directory": "{{ node_exporter_textfile_dir}}"}}]` | Collectors and their config, that should be enabled regardless of the default.                                                     |
```

Comment:
What does
> regardless of the default

mean here?

Reply:
There are collectors that are available but disabled by default.

Reply:
Would `This completely overwrites the default of node_exporter.` explain what you mean, or did I misunderstand?

Reply:
On a per collector basis yes. It does not replace the complete list of defaults.

Reply:
Hm it still is not very clear to me what the description here intends to say.
Would the following be correct?
`Collectors that should be enabled (in addition to the node_exporter defaults) and their config (which overwrites any defaults defined by node_exporter).`

Reply:
Maybe the following would be a suitable description:
`Collectors that will be enabled and their respective configuration. In addition to the collectors defined here there may be others, that are enabled by default.`

Reply:
Yes this sounds good

Reply:
Done.